components/CramerView/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <!-- vim: set tabstop=4 softtabstop=0 expandtab shiftwidth=4 smarttab : -->
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.tibco.ep.sp.contrib</groupId>
+    <artifactId>CramerView</artifactId>
+    <packaging>ep-liveview-fragment</packaging>
+    <version>1.0.0</version>
+    <name>CramerView</name>
+    <description>Illustrates using SMILE operators for correlating sensor readings with failures and presenting a LiveView table of the results for visualizing in TIBCO LiveView Web or TIBCO Spotfire</description>
```

Comment:
This might be too long as this is the description shown in the samples dialog

Reply:
then sample dialog must fix itself and accomodate (scroll, expand bar, etc.)